scripts/bash/run_enceladus.sh

```diff
@@ -284,36 +289,38 @@ set -- "${POSITIONAL[@]}" # restore positional parameters
 # Display values of all declared variables
 #declare -p
 
-# Validation
-VALID="1"
-validate() {
-    if [[ -z "$2" ]]; then
-        echo "Missing mandatory option $1"
-        VALID="0"
+if [ "$HELP_CALL" == "0" ]; then
+    # Validation (only if not help called)
+    VALID="1"
+    validate() {
```

Comment:
Otherwise the helper script doesn't allow to execute the `spark-submit` making the `-help` of limited use.
This way `run_standardization.sh --help` will display the help.